tests/acceptance/owner/not-found-test.js

```diff
@@ -0,0 +1,51 @@
+import Ember from 'ember';
+import { test } from 'qunit';
+import moduleForAcceptance from 'travis/tests/helpers/module-for-acceptance';
+import nonExistentOwnerPage from 'travis/tests/pages/owner/non-existent';
+import { percySnapshot } from 'ember-percy';
+
+let adapterException;
+let loggerError;
+
+moduleForAcceptance('Acceptance | owner/not found', {
+  beforeEach() {
+    // Ignore promise rejection.
+    // Original exception will fail test on promise rejection.
+    adapterException = Ember.Test.adapter.exception;
+    loggerError = Ember.Logger.error;
+    Ember.Test.adapter.exception = () => null;
+    Ember.Logger.error = () => null;
+  },
+
+  afterEach() {
+    Ember.Test.adapter.exception = adapterException;
+    Ember.Logger.error = loggerError;
+  }
```

Comment:
wow, this is quite the technique! I’ll try to remember this, seems useful.

Reply:
I actually just copied this from `repo/not-found-test`, but although it is useful I keep wondering if that's the proper way to do it. Maybe I'll investigate it later.